src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/filters/suppressionsinglefilter/Example1.java

```diff
@@ -15,6 +15,18 @@
   </module>
 </module>
 */
-
+package com.puppycrawl.tools.checkstyle.filters.suppressionsinglefilter;
 // xdoc section -- start
+public class Example1 {
+  // Violation, 'MagicNumberCheck (without suppression)'
+  private static final int MULTIPLIER = 5;
+
+  public int calculate(int value) {
+    return value * MULTIPLIER; // Normally, 5 would trigger MagicNumberCheck
+  }
```

Comment:
Please add MagicNumber to config in header of this file.

All filter configs should be shown with check config, to be exact.

Please update all examples to have Required Checks